h/api/events.py

```diff
@@ -4,9 +4,10 @@
 class AnnotationEvent(object):
     """An event representing an action on an annotation."""
 
-    def __init__(self, request, annotation, action):
+    def __init__(self, request, annotation_id, serialized_annotation, action):
         self.request = request
-        self.annotation = annotation
+        self.annotation_id = annotation_id
+        self.serialized_annotation = serialized_annotation
```

Comment:
Changed it to be `annotation_dict` and made the annotation id a property on the event which fetches it from the dict :+1: